contracts/test/ERC20BridgeTests.t.sol

```diff
@@ -0,0 +1,36 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import {TeleporterTokenBridgeTest} from "./TeleporterTokenBridgeTests.t.sol";
+import {IERC20Bridge} from "../src/interfaces/IERC20Bridge.sol";
+import {SendTokensInput} from "../src/interfaces/ITeleporterTokenBridge.sol";
+import {IERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/utils/SafeERC20.sol";
+
+abstract contract ERC20BridgeTest is TeleporterTokenBridgeTest {
+    using SafeERC20 for IERC20;
+
+    IERC20Bridge public erc20Bridge;
+
+    function testZeroSendAmount() public {
+        vm.expectRevert("SafeERC20TransferFrom: balance not increased");
+        _send(_createDefaultSendTokensInput(), 0);
+    }
+
+    function _send(SendTokensInput memory input, uint256 amount) internal virtual override {
+        erc20Bridge.send(input, amount);
+    }
+
+    function _checkDeposit(uint256 amount) internal virtual override {
```

Comment:
Nit: Is there a more descriptive name than `_checkDeposit` for this helper, and/or could you add a comment explaining it's intended use? If I understand it correctly, may `_setUpExpectedDeposit` is a more accurate name, and we could also and an expected emitted event here I think.

Reply:
added transfer event and renamed functions